presto-blackhole/src/main/java/com/facebook/presto/plugin/blackhole/BlackHoleMetadata.java

```diff
@@ -214,11 +238,19 @@ private PrestoException viewsAreNotSupportedException()
 
     @Override
     public List<ConnectorTableLayoutResult> getTableLayouts(ConnectorSession session,
-            ConnectorTableHandle table,
+            ConnectorTableHandle handle,
             Constraint<ColumnHandle> constraint,
             Optional<Set<ColumnHandle>> desiredColumns)
     {
-        return ImmutableList.of(new ConnectorTableLayoutResult(getTableLayout(session, BLACK_HOLE_TABLE_LAYOUT_HANDLE), TupleDomain.none()));
+        checkNotNull(handle);
```

Comment:
Add a message to the checkNotNull: "handle is null". It makes it easier to debug if we ever encounter that exception.

Also use `Objects.requireNonNull` instead (we're slowly migrating all such usages)